docs/content/docs/dev/table/sql/insert.md

```diff
@@ -208,7 +208,9 @@ column_list:
 **COLUMN LIST**
 
 Given a table T(a INT, b INT, c INT), Flink supports INSERT INTO T(c, b) SELECT x, y FROM S. The expectation is
-that 'x' is written to column 'c' and 'y' is written to column 'b' and 'a' is set to NULL (assuming column 'a' is nullable). 
+that 'x' is written to column 'c' and 'y' is written to column 'b' and 'a' is set to NULL (assuming column 'a' is nullable).
+For the connector developers who want to avoid overwriting non-target columns with null values when processing partial column updates,
```

Comment:
It seems like we miss how the connector devs can avoid it? may be some like:
```
For the connector .... when processing partial column updates, they can find the user-specified target column list and only update the target columns.
```
?

Reply:
How about: "For connector developers who want to avoid overwriting non-target columns with null values when processing partial column updates, you can get the information about the target columns specified by the user's insert statement from 'getTargetColumns(link...)' and decide how to process the partial updates."

Reply:
The corresponding chinese version: 连接器开发人员在处理部分列更新时，如果希望避免用空值覆盖非目标列，可以从 "getTargetColumns(link...) "中获取用户插入语句指定的目标列信息，然后决定如何处理部分更新。

Reply:
Thanks, I like the tweaked description.